app/models/user.rb

```diff
@@ -32,6 +32,10 @@ def superadmin_or_department_manager?
     superadmin? || department_manager?
   end
 
+  def department?
```

Comment:
this is what the linter thinks about `has_department?`
> [Style/PredicateName] Style/PredicateName: Rename `has_department?` to `department?`.

But `with_department?` doesn't seem to upset it.
`manager_with_department?` is in my view too long and `@user.manager_with_department?` is doublemeaning, because as far as I can see a `@user` should only be assignable to departments if `@user.role == 'department_manager'`. But `with_department?` looks quite all right to me. What do you guys think?